language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FunctionTypeDescriptorNodeContext.java

```diff
@@ -76,12 +81,37 @@ public List<LSCompletionItem> getCompletions(BallerinaCompletionContext context,
             ruleContext = RuleContext.PARAMETER_CTX;
         } else if (this.withinReturnKWContext(context, node)) {
             completionItems.add(new SnippetCompletionItem(context, Snippet.KW_RETURNS.get()));
+        } else if (node.parent().kind() == SyntaxKind.OBJECT_FIELD) {
+            SnippetCompletionItem initFuncCompletionItem =
```

Comment:
Let's add a comment specifying the completion context

Eg: 
```java
 /**covers 
object {
    function <cursor> 
 }
*/

Reply:
Added